src/function.cc

```diff
@@ -325,7 +324,7 @@ Local<Value> FunctionCall (
      * and for error, if it can throw
      */
 
-    GIArgument total_arg_values[func->n_total_args];
+    GIArgument *total_arg_values = new GIArgument[func->n_total_args]();
```

Comment:
Oh right we were using [this form](https://github.com/romgrk/node-gtk/blob/56335f82387011ecb590a8bffe88e35c7cb27d02/src/closure.cc#L50-L54) to deal with this. Can we do the same here? `FunctionCall` runs on every function call (who would have guessed?) so I'd rather avoid the allocation if possible.